src/controllers/tripsController.js

```diff
@@ -225,6 +275,30 @@ export default class requestController {
           role
         }],
       });
+
+      const managerInfo = await db.User.findOne({ where: { id: user.managerId } });
+
+      const notification = await notifService.createNotif(
+        newMulticityRequest.managerId,
+        managerInfo.email,
+        `A trip request to ${newMulticityRequest.destination} on ${newMulticityRequest.departureDate} has been requested by ${user.firstName}, ${user.lastName}, it is waiting your approval.`,
+        '#'
+      );
+      const content = {
+        intro: req.__(notification.content),
+        instruction: req.__(`To view this ${newMulticityRequest.status} request you made click below`),
```

Comment:
is the internalization working for those strings? if not could you add it so that when a user changes the language to french, those string also changes to french